### ISSUE TITLE
Add image alt text to staff image

### ISSUE THREAD

**Issue by JoeProgrammer88** [SpeakingInBits/PC2#272]
On the `/About` page, the third staff member does not have her name in the alt text of the photo. Add Cary Vazquez to the alt text
